src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -1116,6 +1116,13 @@ public SolutionState AddAnalyzerReference(ProjectId projectId, AnalyzerReference
                 throw new ArgumentNullException(nameof(analyzerReference));
             }
 
+            //PROTOTYPE
```

Comment:
Since you have to touch this anyways, want to just update AddAnalyzerReference to call into AddAnalyzerReferences and avoid the duplication? I suspect the history here is that AddAnalyzerReference**s** was added later and for some reason we just didn't update this to call into it.